server/controllers/stock/index.js

```diff
@@ -532,24 +532,29 @@ async function listLotsDepot(req, res, next) {
     params.defaultPeriodEntry = params.defaultPeriod;
     delete params.defaultPeriod;
   }
+
   try {
     const data = await core.getLotsDepot(null, params);
 
     const queryTags = `
       SELECT BUID(t.uuid) uuid, t.name, t.color, BUID(lt.lot_uuid) lot_uuid
```

Comment:
Should we be using the SQL keyword 'AS' to define column aliases here?  I know it is optional, but shouldn't we be consistent for readability?  I personally prefer adding the AS since it makes it more readable to me.

Reply:
I agree and  I generally try to.  Do you think it's worth trying to enforce such a style for PRs?

Reply:
I personally think it is a good idea both for readability.   À mon avis!  However, we already enforce various Javascript niceties via hound.  If hound can sniff SQL and check this, then sure, I think announcing that and enforcing it via hound makes sense and is not too painful (I assume hound only checks changed files).   So that developers only need to change future behavior.